data-import-from-s3/src/main/java/no/unit/nva/dataimport/DataImportHandler.java

```diff
@@ -108,16 +108,16 @@ private String requestToJson(Map<String, String> request) {
         return failedImports;
     }
 
-    private ParallelMapper<String, BatchWriteItemResult> insertFilesToDynamo(List<String> filesToInsert)
+    private ParallelMapper<UnixPath, BatchWriteItemResult> insertFilesToDynamo(List<UnixPath> filesToInsert)
         throws InterruptedException {
         return new ParallelMapper<>(filesToInsert, this::insertFileToDynamo).map();
     }
 
-    private List<String> extractFilenamesFromFailedImports(
+    private List<UnixPath> extractFilenamesFromFailedImports(
         List<ImportResult<FailedDynamoEntriesReport>> failedImports) {
         return failedImports.stream().parallel()
                    .map(ImportResult::getInput)
-                   .map(FailedDynamoEntriesReport::getInputFilename)
+                   .map(FailedDynamoEntriesReport::getInputFilePath)
```

Comment:
getinputFilename was incorrect because it returns the whole path(before as String, now as UnixPath)